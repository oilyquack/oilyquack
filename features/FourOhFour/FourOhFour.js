import { Link } from "gatsby";
import React from "react";

import david from "./assets/davidbyrne.gif";
import styles from "./FourOhFour.module.css";

const FourOhFour = () => (
  <>
    <h1>FOUR, OH FOUR!</h1>
    <p>
      <Link to="/">Home</Link> is where you want to be.
    </p>
    <div className={styles.content}>
      <img alt="David Byrne" className={styles.image} src={david} />

      <span className={styles.quote}>
        <p>And you may ask yourself</p>
        <p>How do I work this?</p>
        <p>And you may ask yourself</p>
        <p>Where is that large automobile?</p>
        <p>And you may tell yourself</p>
        <p>This is not my beautiful house!</p>
        <p>And you may tell yourself</p>
        <p>This is not my beautiful wife!</p>

        <p className={styles.author}>â€” David Byrne, 1980</p>
      </span>
    </div>
  </>
);

export default FourOhFour;
