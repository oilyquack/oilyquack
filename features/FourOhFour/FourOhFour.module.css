.content {
  display: flex;
  flex-direction: column;
}

.image {
  height: 100%;
  width: 100%;
}

.author {
  font-style: italic;
}
@media (min-width: 768px) {
  .content {
    flex-direction: row;
  }

  .image {
    width: 45%;
  }

  .quote {
    padding: 0 30px;
  }

  .quote :nth-child(1) {
    margin-top: 0;
  }
}
