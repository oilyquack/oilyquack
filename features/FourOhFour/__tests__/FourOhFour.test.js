import { render } from "@testing-library/react";
import React from "react";

import FourOhFour from "../FourOhFour";

describe("features : FourOhFour", () => {
  it("renders correctly", () => {
    const { container } = render(<FourOhFour />);
    expect(container).toMatchSnapshot();
  });
});
