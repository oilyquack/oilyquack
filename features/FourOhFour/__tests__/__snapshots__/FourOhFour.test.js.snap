// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`features : FourOhFour renders correctly 1`] = `
<div>
  <h1>
    FOUR, OH FOUR!
  </h1>
  <p>
    <a
      href="/"
    >
      Home
    </a>
     is where you want to be.
  </p>
  <div
    class="content"
  >
    <img
      alt="David Byrne"
      class="image"
      src="test-file-stub"
    />
    <span
      class="quote"
    >
      <p>
        And you may ask yourself
      </p>
      <p>
        How do I work this?
      </p>
      <p>
        And you may ask yourself
      </p>
      <p>
        Where is that large automobile?
      </p>
      <p>
        And you may tell yourself
      </p>
      <p>
        This is not my beautiful house!
      </p>
      <p>
        And you may tell yourself
      </p>
      <p>
        This is not my beautiful wife!
      </p>
      <p
        class="author"
      >
        â€” David Byrne, 1980
      </p>
    </span>
  </div>
</div>
`;
